plugins/00_dokku-standard/commands

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
-source "$PLUGIN_PATH/common/functions"
-source "$PLUGIN_PATH/config/functions"
+source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+source "$PLUGIN_CORE_AVAILABLE_PATH/config/functions"
```

Comment:
If I change this to `source "$PLUGIN_AVAILABLE_PATH/config/functions`, will that pick up whatever config plugin is available? Basically, I'd like to be able to override every core plugin except common.